BUILDING.md

```diff
@@ -105,7 +105,11 @@ You need to install the following
 - Xcode
 - [Doxygen](#doxygen) (only if generating documentation)
 
-For the load tests applications you need to install the [Vulkan SDK](#vulkan-sdk).
+For the load tests applications you need to install the Vulkan SDK.
+To build for macOS you need set the CMake cache variable `VULKAN_SDK` to the root of the Vulkan SDK, if it is not already set.
+To build for iOS you need to set the CMake cache variable `MOLTEN_VK_SDK` to the root of MoltenVK inside the Vulkan SDK, if it is not already set.
```

Comment:
Add the following after this line:

Caution: `setup.env` in the macOS Vulkan SDK sets VULKAN_SDK to the macOS folder of the SDK, a sibling of the MoltenVK folder.